cpp/src/tracing_dummy.cpp

```diff
@@ -0,0 +1,54 @@
+/*
```

Comment:
This file should be renamed as tracing_stub.cpp to be in line with naming in other places.